pkgs/by-name/ty/typos/package.nix

```
@@ -17,6 +20,17 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-WuYAE2msctMf+7C1h5pNGyjrEInnT6PxfZ94bZhzB5w=";
 
+  passthru = {
+    tests.version = testers.testVersion {
+      inherit version;
+
+      package = typos;
+      command = "typos --version";
+    };
```

Unless it was failing without making those changes, that is the default behaviour for the version tester https://github.com/NixOS/nixpkgs/blob/8ebd7303198eb72927b8891bf22c28dbdda8bcd1/doc/build-helpers/testers.chapter.md?plain=1#L179
```suggestion
    tests.version = testers.testVersion { };
```

---

@kachick sorry for the orginal typo, i updated the comment but you had updated by then. it should be 
```nix
tests.version = testers.testVersion { package = typos;  };
```
Which specifies the package. Terribly sorry.

---

Thank you for the polite response :pray: I agreed and applied your suggestion in https://github.com/NixOS/nixpkgs/pull/354982/commits/1f983dbc47767a90b62f4e64fae6284d8b75a3ab